redfish_client/connector.py

```diff
@@ -51,7 +51,11 @@ def _url(self, path):
 
     def _request(self, method, path, payload=None):
         args = dict(json=payload) if payload else {}
-        resp = self._client.request(method, self._url(path), **args)
+        try:
+            resp = self._client.request(method, self._url(path), **args)
+        except requests.exceptions.ConnectionError:
+            raise InaccessibleException(
+                'Endpoint at {} is not accessible'.format(self._base_url))
```

Comment:
Nitpick: We should use `"` in place of `'` for consistency with the rest of the code base.